R/ms_chart.R

```diff
@@ -403,10 +441,84 @@ format.ms_chart <- function(x, id_x, id_y, sheetname = "sheet1", drop_ext_data =
   y_axis_str <- sprintf("<%s>%s</%s>", x$axis_tag$y, y_axis_str, x$axis_tag$y)
 
 
+  secondary_x_y <- TRUE
+  if (length(x$secondary)) {
+
+    ser_id <- length(x$yvar) + 1L
+
+    for (sec in seq_along(x$secondary)) {
+
+      # is_sec_x <- attr(x$secondary[[sec]], "secondary_x")
+      is_sec_y <- attr(x$secondary[[sec]], "secondary_y")
+
+      if (is_sec_y) {
+        x_id <- as.character(67917199)
+        y_id <- as.character(320476559)
+      } else {
+        x_id <- id_x
+        y_id <- id_y
+      }
```

Comment:
These ids point to the chart to the axis. If we are on the primary x and y axis, we use the default values, otherwise the values for the secondary x and y axis. Probably we can have a secondary y axis on top as well, but I have never required any.